app/components/Felt/Pot.tsx

```diff
@@ -37,105 +37,150 @@ const Pot = () => {
-                                                        base: '10px',
-                                                    }}
-                                                    color={'white'}
-                                                >
-                                                    total{' '}
-                                                    <Text
-                                                        as={'span'}
-                                                        color={'white'}
-                                                        fontWeight={'medium'}
-                                                        fontSize={{
-                                                            xl: '17px',
-                                                            lg: '16px',
-                                                            md: '14px',
-                                                            base: '13px',
-                                                        }}
-                                                    >
-                                                        {
-                                                            game.pots[index]
-                                                                .amount
-                                                        }
-                                                    </Text>
-                                                </Text>
-                                            </Box>
-                                        ) : (
-                                            <Text>&nbsp;</Text>
-                                        )}
-                                    </Flex>
-                                    <Flex
-                                        paddingX={'14'}
-                                        justifyContent={'center'}
-                                        mb={'10%'}
-                                    >
-                                        <Text
-                                            fontSize={{
-                                                xl: '17px',
-                                                lg: '16px',
-                                                md: '12px',
-                                                base: '9px',
-                                            }}
-                                            color={'white'}
-                                            fontWeight={'bold'}
-                                        >
-                                            {pot.amount}
-                                        </Text>
-                                    </Flex>
-                                </Flex>
-                            </Flex>
-                        );
-                    }
-                })}
-            </Flex>
-        );
+    if (!isGameRunning) {
+        return null;
     }
 
-    return null;
+    const hasTotal = game.pots[0]?.amount > 0;
```

Comment:
this seems like a bad implementation as it only handles 1 sidepot. Theoretically we can have more so we should be able to handle all of them. This means we need a solution to handle the whole array. We should display the main one as we currently do. Any sidepot should be a small bubble above the main one and be formated as "SP{#}: Amount"